public/app/features/dashboard/components/AnnotationSettings/AnnotationSettingsEdit.tsx

```diff
@@ -78,6 +130,20 @@ export const AnnotationSettingsEdit = ({ editIdx, dashboard }: Props) => {
   };
 
   const isNewAnnotation = annotation.name === newAnnotationName;
+  const formWidth = 50;
+
+  const panels: Array<SelectableValue<number>> = useMemo(
+    () =>
+      dashboard?.panels
+        .map((panel) => ({
+          value: panel.id,
+          label: panel.title ?? `Panel ${panel.id}`,
+          description: panel.description,
+          imgUrl: config.panels[panel.type].info.logos.small,
+        }))
+        .sort((a, b) => (a.label > b.label ? 1 : -1)) ?? [],
+    [dashboard]
+  );
```

Comment:
a way to move this to DashboardModel and maybe we can reuse this in the Shared dashboard query editor?

Reply:
It looks a bit different, hm..